src/Behat/Testwork/Tester/ServiceContainer/TesterExtension.php

```diff
@@ -109,7 +109,7 @@ public function load(ContainerBuilder $container, array $config)
         $this->loadSpecificationTester($container);
     }
 
-    public function process(ContainerBuilder $container)
+    public function process(ContainerBuilder $container): void
```

Comment:
Also a potential BC break - `TesterExtension` is abstract and theoretically intended as a base class for other testers on the "Testwork" framework though I'm not aware of projects outside Behat using it.